pkgs/by-name/sw/switcheroo-control/package.nix

```diff
@@ -28,6 +29,7 @@ python3Packages.buildPythonApplication rec {
     ninja
     meson
     pkg-config
+    wrapGAppsNoGuiHook
   ];
 
```

Comment:
Please add 

```
  dontWrapGApps = true;

  # Arguments to be passed to `makeWrapper`, only used by buildPython*
  preFixup = ''
    makeWrapperArgs+=("''${gappsWrapperArgs[@]}")
  '';
```

To avoid double wrapping.